.bingo/Variables.mk

```diff
@@ -26,25 +26,19 @@ GOLANGCI_LINT := $(GOBIN)/golangci-lint-v1.27.0
 $(GOLANGCI_LINT): .bingo/golangci-lint.mod
 	@# Install binary/ries using Go 1.14+ build command. This is using bwplotka/bingo-controlled, separate go module with pinned dependencies.
 	@echo "(re)installing $(GOBIN)/golangci-lint-v1.27.0"
-	@cd .bingo && $(GO) build -mod=mod  -modfile=golangci-lint.mod -o=$(GOBIN)/golangci-lint-v1.27.0 "github.com/golangci/golangci-lint/cmd/golangci-lint"
-
-JUNITREPORT := $(GOBIN)/junitreport
-$(JUNITREPORT): .bingo/junitreport.mod
-	@# Install binary/ries using Go 1.14+ build command. This is using bwplotka/bingo-controlled, separate go module with pinned dependencies.
-	@echo "(re)installing $(GOBIN)/junitreport"
-	@cd .bingo && $(GO) build -mod=mod -modfile=junitreport.mod -o=$(GOBIN)/junitreport "github.com/openshift/origin/tools/junitreport"
```

Comment:
This should not be deleted, although Bingo has an issue everytime doing this delete.